Plugin.php

```diff
@@ -14,19 +14,19 @@ class Plugin extends PluginBase
     public function pluginDetails()
     {
         return [
-            'name'        => 'rainlab.builder::lang.plugin.name',
```

Comment:
Turn off your editor's auto formatting and try this again please, all of these formatting changes won't be accepted.

Reply:
Excuse me. Sent again.